movement/movement.c

```diff
@@ -23,6 +23,8 @@
  */
 
 #define MOVEMENT_LONG_PRESS_TICKS 64
+#define DEBOUNCE_TICKS_DOWN 0  // In terms of *7.8125ms
+#define DEBOUNCE_TICKS_UP   0  // In terms of *7.8125ms
```

Comment:
We should also add comments in the code itself showing how to arrive at this figure. It was calculated using the current 128 Hz interrupt but this could change in the future: I've seen high resolution timer work aiming at multiplexing the hardware timers and freeing up the 128 Hz tick for use by applications.

I think we also need detailed documentation here on why it's not recommended that the user set the values to 1. I'm not sure I understood the reason when I read the PR description.